block-explorer/elasticsearch/mappings/block.json

```diff
@@ -0,0 +1,38 @@
+{
+  "mappings": {
+    "dynamic": false,
+    "properties": {
```

Comment:
Like in transaction, do we need `timestamp` field?

Reply:
I would leave it for now